site/app/libraries/FileUtils.php

```diff
@@ -502,32 +514,62 @@ public static function validateUploadedFiles($files) {
         for ($i = 0; $i < $num_files; $i++) {
             //extract the values from each file
             $name = $files['name'][$i];
-            $type = mime_content_type($files['tmp_name'][$i]);
-            $size = $files['size'][$i];
-            $err_msg = "";
+            $tmp_name = $files['tmp_name'][$i];
+            $type = mime_content_type($tmp_name);
+            
+            $zip_status = FileUtils::getZipFileStatus($tmp_name);
+            $errors = [ErrorMessages::uploadErrors($files['error'][$i])];
```

Comment:
Let's make this:
```
$errors = [];
if ($files['error'][$i] !== UPLOAD_ERR_OK) {
    $errors[] = ErrorMessages::uploadErrors($files['error'][$i]);
}
```
Let's only append to `$errors` if there's an actual error message.